pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -330,3 +343,20 @@ func (r *ReconcileCertificateRequest) revokeCertificateAndDeleteSecret(reqLogger
 	}
 	return nil
 }
+
+// relocationBailOut checks to see if there's a cluster relocation in progress
+func relocationBailOut(k client.Client, nsn types.NamespacedName) (relocating bool, err error) {
+	relocating = false
+
+	cd := &hivev1.ClusterDeployment{}
+	err = k.Get(context.TODO(), nsn, cd)
+
```

Comment:
I get that the loop below correctly iterates zero times if the `Get` failed, but it would be more defensive and explicit to include

```go
if err != nil {
    return
}
```

Reply:
fair enough, clarity is always a good idea